sopel/bot.py

```diff
@@ -519,8 +531,7 @@ def call(self, func, sopel, trigger):
                 self._times[trigger.sender][func] = current_time
 
     def dispatch(self, pretrigger):
-        """Match an incoming, parsed message from the server and dispatch it to any registered
-        callables.
+        """Dispatch a parsed message to any registered callables
```

Comment:
I think we're trying to be consistent about having `.` at the end of these short ones? Maybe?

Even if we aren't yet, we probably should be.

Reply:
That one slipped through - added.